requirements.txt

```diff
@@ -5,3 +5,4 @@ lxml
 pytest
 pytest-cov
 pyutilib
+cpp-demangle
```

Comment:
From https://pypi.org/project/cpp-demangle/

```
This is mainly an experiment in creating python extensions in Rust.
```
Is there are more stable lib for this? I saw that https://pypi.org/project/cxxfilt/ is also alpha.

Reply:
You can also use the command `c++filt` to get the demangle names.